tests/test_modbus_block.py

```diff
@@ -94,7 +100,7 @@ def test_exception_code(self, mock_client):
                          'are not allowed or do not exist in slave')
         blk.stop()
 
-    @patch('modbus.modbus_block.sleep')
```

Comment:
This is what was causing it to fail. If the modbus block was in a `blocks` directory with an `__init__.py` for example, the module `modbus` doesn't exist, but rather `blocks.modbus`.